BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.kt

```diff
@@ -64,38 +57,19 @@ internal class AnalyticsClient @VisibleForTesting constructor(
         )
     }
 
-    fun writeAnalytics(inputData: Data): ListenableWorker.Result {
-        val eventName = inputData.getString(WORK_INPUT_KEY_EVENT_NAME)
-        val payPalContextId = inputData.getString(WORK_INPUT_KEY_PAYPAL_CONTEXT_ID)
-        val linkType = inputData.getString(WORK_INPUT_KEY_LINK_TYPE)
-        val timestamp = inputData.getLong(WORK_INPUT_KEY_TIMESTAMP, INVALID_TIMESTAMP)
-        val venmoInstalled = inputData.getBoolean(WORK_INPUT_KEY_VENMO_INSTALLED, false)
-        val isVaultRequest = inputData.getBoolean(WORK_INPUT_KEY_IS_VAULT_REQUEST, false)
-        val startTime = inputData.getLong(WORK_INPUT_KEY_START_TIME, INVALID_TIMESTAMP)
-        val endTime = inputData.getLong(WORK_INPUT_KEY_END_TIME, INVALID_TIMESTAMP)
-        val endpoint = inputData.getString(WORK_INPUT_KEY_ENDPOINT)
-
-        return if (eventName == null || timestamp == INVALID_TIMESTAMP) {
+    fun performAnalyticsWrite(inputData: Data): ListenableWorker.Result {
+        val analyticsJSON = inputData.getString(WORK_INPUT_KEY_ANALYTICS_JSON)
+        return if (analyticsJSON == null) {
```

Comment:
We are losing the INVALID_TIMESTAMP check here - is that intentional?

Reply:
Yeah I think we needed `INVALID_TIMESTAMP` because of WorkManager's `Data` class. This PR increases our alignment with the [iOS implementation](https://github.com/braintree/braintree_ios/blob/d93bd064ad5b333b128d50bbf397ef45a42a8ab1/Sources/BraintreeCore/Analytics/FPTIBatchData.swift#L52).